### PR TITLE
improve(docs): Add guide for installing Infra with kind

### PR DESCRIPTION
**Pull request by pdevine**
## Summary

Create a guide which explains how to install Infra with kind

## Checklist

- [x] Updated associated docs where necessary

### CODE REVIEW

File: docs/install-infra-on-kind.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Installing Infra on kind
+
+This guide will show you how to set up [kind](https://kind.sigs.k8s.io/) and Infra in Linux
```
Comment (BruceMacD) [inline docs/install-infra-on-kind.md:3]:
```suggestion
This guide will show you how to set up [kind](https://kind.sigs.k8s.io/) and Infra on Linux
```

File: docs/install-infra-on-kind.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+using the Metallb load balancer. Docker Desktop for macOS and Windows both use a virtual machine
+which runs Linux, and does not directly expose docker's network to the host. Consider using
+Docker's built-in kubernetes implementation instead of kind if you are using Docker Desktop.
+
+This guide assumes that you have already [installed kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) and [docker](https://docs.docker.com/engine/install/).
+
+### 1. Download the latest version of kind
+
+You'll first need to download a copy of kind. We'll use one of the pre-compiled binaries,
```
Comment (BruceMacD) [inline docs/install-infra-on-kind.md:12]:
```suggestion
You'll first need to download kind. We'll use one of the pre-compiled binaries,
```

**Review by BruceMacD — COMMENTED** [review]
Looks good, I think this document should be in the `docs/guides` directory though

**Review by dnephin — COMMENTED** [review]
Nice! I did this yesterday and it mostly worked. I think I didn't do the step with setting IP addresses correctly, which is likely what wasn't working for me.

I like Bruce's suggestions, but otherwise LGTM.

**Comment by pdevine** [comment]
Going to close this in favour of posting it in the Discussions forum for now.

**Pull request closed without merging**